planning/autoware_trajectory_optimizer/include/autoware/trajectory_optimizer/trajectory_optimizer_plugins/plugin_utils/trajectory_point_fixer_utils.hpp

```diff
@@ -109,6 +109,28 @@ void resample_single_cluster(
 void resample_close_proximity_points(
   TrajectoryPoints & traj_points, const Odometry & current_odometry, const double min_dist_m);
 
+/**
+ * @brief Removes invalid points from the trajectory.
+ *
+ * Removes points with NaN or Inf values from the trajectory. Validates each point
+ * using validate_point() from common utils.
+ *
+ * @param input_trajectory The trajectory points to be cleaned (modified in place)
+ */
+void remove_invalid_points(TrajectoryPoints & input_trajectory);
+
+/**
+ * @brief Removes points from the trajectory that are too close to each other.
+ *
+ * Iterates through the trajectory and removes consecutive points that are closer
+ * than the minimum distance threshold. Always keeps the first point.
+ *
+ * @param input_trajectory_array The trajectory points to be cleaned (modified in place)
+ * @param min_dist The minimum distance between points (default: 1E-2)
+ */
+void remove_close_proximity_points(
+  TrajectoryPoints & input_trajectory_array, const double min_dist = 1E-2);
```

Comment:
nits:
```suggestion
  TrajectoryPoints & input_trajectory_array, const double min_dist = 1e-2);
```
Seems autoware uses both `E` and `e`, but I think `e` is generally used.

Reply:
DOne